doc/netplan.md

```diff
@@ -585,7 +585,8 @@ eth1:
 ``routes`` (mapping)
 
 :    The ``routes`` block defines standard static routes for an interface.
-     At least ``to`` and ``via`` must be specified.
+     At least ``to`` and ``via`` must be specified.  Routes with scope
+     ``link`` does not need to specify ``via``.
```

Comment:
I feel like keeping the first sentence as-is might be deserving, as someone might overlook the latter sentence contradicting it. What do you think of this alternative?

```suggestion
     At least ``to`` must be specified.  Unless the scope is ``link``, ``via`` is
     also required.
```

Reply:
(took a while until I understood "deserving" should be "deceiving" :smile: )

Yes. I think your suggestion is better.  May still be slightly misleading (and this was introduced by me), since the keyword controlling this is not `scope: link`, but `on-link: true`.

```suggestion
     At least ``to`` must be specified.  Unless route is ``on-link``, ``via`` is
     also required.
```

Reply:
Actually, I meant "disserving" but my brains and my fingers apparently disagreed ;-).
Your latest suggestion is indeed much better than mine, but I'll have to check if `scope: link` doesn't have the same effect, as I believe it should?

Reply:
I'm sorry, I confused myself.  Indeed it is `scope: link` which is correct.  `on-link: true` does not work.